src/Common/Config/ConfigProcessor.cpp

```diff
@@ -234,7 +234,7 @@ void ConfigProcessor::merge(XMLDocumentPtr config, XMLDocumentPtr with)
 
 static std::string layerFromHost()
 {
-    utsname buf;
+    struct utsname buf;
```

Comment:
I am actually curious how this code compiles under other platforms. The type `struct utsname` is almost universally _not_ typedef'd to just `utsname`.